components/sections/Title.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <section
+    class="flex flex-col justify-center items-center base-wrapper mt-20 md:mt-28"
+  >
+    <BaseHeading size="h1" class="text-primary-dark">{{
+      block.title
+    }}</BaseHeading>
+
+    <BaseHeading
+      size="h3"
+      class="text-secondary-dark text-center uppercase font-bold -mt-4"
+      >{{ block.heading }}</BaseHeading
+    >
```

Comment:
This is one of the conditional fields. we should proably add `v-if="block.heading` as an attribute